remix/app/lib/toast/makeToast.ts

```diff
@@ -0,0 +1,11 @@
+import toast from 'react-hot-toast';
+
+export default function makeToast(toastMode: string, toastMessage: string) {
+  if (toastMode === 'success') {
```

Comment:
How about using a switch statement here instead 💡

Reply:
Yes, `switch statement` would be better than `if/else` here, but that's I thought my use-case will be only `success`, `error`, and `not provide toastMode`.

But if look at this from the scaling perspective `switch statement` will be better, as we don't use what's gonna come in the future.

I will Improve it.

Reply:
Improved here: [7ba7a3b](https://github.com/nimblehq/js-framework-benchmark/pull/120/commits/7ba7a3b2c928919163e49e2c82434c454d3bbe95)